.github/workflows/release-publish-releasenotes-twitter.yml

```diff
@@ -2,8 +2,8 @@
 # GitHub Action to Publish Tweet of release notes.
 #
 #############################################################################
-name: "R: Publish Tweet"
-run-name: "R: Publish Tweet ${{ inputs.version }}"
+name: "R: Publish Releasenotes Twitter"
+run-name: "R: Publish Releasenotes Twitter ${{ inputs.version }}"
```

Comment:
I should change this to `R: Publish Release Notes Twitter` to stay consistent with `R: Update xdoc with Release Notes`

Reply:
Good

Reply:
Done.